tests/integration/summary_plot/test_summary_plot_integration.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime
```

Comment:
We both omitted having an `__init__.py` in these folders, me in `integration` and you here, could you add them both? That gives some `pylint` errors, other than that this looks good now 👍